src/main/java/net/dries007/tfc/config/ServerConfig.java

```diff
@@ -16,7 +15,7 @@
 import net.dries007.tfc.config.animals.ProducingMammalConfig;
 import net.dries007.tfc.util.Alloy;
 
-import static net.dries007.tfc.TerraFirmaCraft.MOD_ID;
+import static net.dries007.tfc.TerraFirmaCraft.*;
```

Comment:
This is a result of the import rule changes I made in 7df3754be0860c4760a57ddd0cc506a024be1256. I didn't apply it to the whole project (because ew + lazy) so this will just slowly standardize as files are edited. Essentially:
```java
import anything; // no wildcards

import tfc;

import static .*; 
```

I explicitly set wildcards to always/allowed because of using them for assertions i.e. `import static org.junit.Assertions.*`